/var/tmp/portage/sys-fs/libeatmydata-130/work/libeatmydata-130/libeatmydata/test/tst-invalidfd.c:31:13: error: implicit declaration of function ‘sync_file_range’; did you mean ‘copy_file_range’? [-Werror=implicit-function-declaration]
   31 |         r = sync_file_range(fd, 0, 0, 0);
      |             ^~~~~~~~~~~~~~~
      |             copy_file_range
--- a/libeatmydata/test/tst-invalidfd.c
+++ b/libeatmydata/test/tst-invalidfd.c
@@ -17,6 +17,11 @@
 
 #include <stdio.h>
 #include <errno.h>
+
+#ifdef HAVE_SYNC_FILE_RANGE
+#include <fcntl.h>
+#endif
+
 #include <unistd.h>
 #include <assert.h>
 
